src/components/SampleDistributionOLSEstimators/SlopeDistributionPlot.js

```diff
@@ -3,22 +3,37 @@ import { max, min } from 'mathjs';
 import PropTypes from 'prop-types';
 import { olsSampleType } from '../../lib/types.js';
 import { getCounts } from '../../lib/stats-utils.js';
-import { OLSE_VALUES } from '../../lib/constants.js';
+import { Form } from 'react-bootstrap';
+import { useState } from 'react';
+import StandardNormalOLS from './StandardNormalOLS.js';
 
-export default function SlopeDistributionPlot({ samples, populationShape }) {
+export default function SlopeDistributionPlot({ samples }) {
   const plotData = getCounts(samples.map(({ slope }) => slope));
+  const [standardized, setStandardized] = useState(false);
+
+  let plot = <DotPlot
+  series={[{ name: 'slopes', data: plotData }]}
+  title="Distribution of Sample Slopes"
+  xMin={min(-5, ...plotData.map(({ x }) => x))}
+  xMax={max(5, ...plotData.map(({ x }) => x))}
+  yMax={max(4, ...plotData.map(({ y }) => y))}
+  xLabel="Slope"
+/> 
+
+if (standardized) {
+  plot = <StandardNormalOLS samples={samples}
+  sampleSize={+samples || 1} />
```

Comment:
this doesn't make sense, `samples` is a list

Reply:
So we don't need sampleSize, right?

Reply:
Are you using it in `StandardNormalOLS`?